src/insertColonCommand.ts

```diff
@@ -23,6 +24,12 @@ const cssDataProvider = getDefaultCSSDataProvider();
 const properties = cssDataProvider.provideProperties();
 const allCSSFunctions = getCSSFunctions();
 
+const normalizeRegex = (regex: string) => {
+  return regex
+    .replace(/\[_\$\[:alpha:]]\[_\$\[:alnum:]]/g, "[a-zA-Z0-9$_]")
+    .replace(/\[_\$\[:alpha:]]\[_\$\\\.\[:alnum:]]/g, "[a-zA-Z0-9$_.]");
```

Comment:
Am I right in thinking this is just unescaping the Regex being passed in?

Reply:
If that is the case I would leave a comment here explaining why we need to do this

Reply:
Oh right ok I see what you’re saying. Yeah I think this may be a better path than bringing in oniguruma just as long as it’s maintainable and we’re not adding more and more to it overtime. I still think a comment here is useful